Feature/440 re enabled parquet import in notebook first steps.ipynb (#445)
exasol/ai-lab: README.md

The Exasol AI Lab is a pre-configured container designed to empower data scientists. It streamlines common data science and AI tasks, including data loading, preparation, exploration, model training, and deployment. Whether you’re a seasoned practitioner or just getting started, the AI Lab provides a hassle-free experience.

![Screenshot Transformers Extension](https://github.com/exasol/ai-lab/blob/4.0.0/doc/user_guide/ai-lab-screenshot.png)

Key Features:

> * **Jupyter Notebook Environment**: The heart of the AI Lab is a robust Jupyter Notebook environment. It’s where you’ll work on your data science projects.

Reviewer: ```suggestion
* **Jupyter Notebook Environment**: The heart of the AI Lab is a robust Jupyter Notebook environment. It is where you will work on your AI and Data Science projects.
```
